### PR TITLE
Morgan logging middleware, commenting out supercronic for now

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,17 +11,17 @@ ADD / /cortex/
...
-  SUPERCRONIC=supercronic-linux-amd64 \
-  SUPERCRONIC_SHA1SUM=be43e64c45acd6ec4fce5831e03759c89676a0ea
-
-# install supercronic
-RUN curl -fsSLO "$SUPERCRONIC_URL" \
-  && echo "${SUPERCRONIC_SHA1SUM}  ${SUPERCRONIC}" | sha1sum -c - \
-  && chmod +x "$SUPERCRONIC" \
-  && mv "$SUPERCRONIC" "/usr/local/bin/${SUPERCRONIC}" \
-  && ln -s "/usr/local/bin/${SUPERCRONIC}" /usr/local/bin/supercronic
+# # set supercronic config
```
Comment (craigspaeth) [inline Dockerfile]: We have version control so just feel free to delete this.

File: index.js [hunk 2]
```diff
@@ -37,7 +40,7 @@ export const main = async () => {
 
   // Start server
   return app.listen(PORT, () => {
-    console.log(`🧠  The neurons are firing: http://localhost:${PORT}`)
+    console.log(`🧠  The neurons are firing on port: ${PORT}`)
```
Comment (craigspaeth) [inline index.js]: 👍

**Comment by craigspaeth** [comment]
LGTM but I'd prefer we just delete instead of comment out.

**Comment by lionsdwarf** [comment]
👍 Will do.

**Pull request merged**